Include/import.h

```diff
@@ -38,11 +38,26 @@ PyAPI_FUNC(PyObject *) PyImport_ExecCodeModuleObject(
     );
 #endif
 PyAPI_FUNC(PyObject *) PyImport_GetModuleDict(void);
+#ifndef Py_LIMITED_API
+PyAPI_FUNC(int) _PyImport_IsInitialized(PyInterpreterState *);
+#endif
+PyAPI_FUNC(PyObject *) PyImport_GetModule(PyObject *name);
+#ifndef Py_LIMITED_API
+PyAPI_FUNC(PyObject *) _PyImport_GetModule(PyObject *name);
+PyAPI_FUNC(PyObject *) _PyImport_GetModuleWithError(PyObject *name);
+PyAPI_FUNC(PyObject *) _PyImport_GetModuleString(const char *name);
```

Comment:
I don't tink that it's worth it to add _PyImport_GetModuleString(): it's only used 2 times. Please use _PyImport_GetModuleId() with_Py_IDENTIFIER, and remove _PyImport_GetModuleString(). I dislike having N versions of the same function.

Reply:
Good point.  I'll drop it.